qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/VisitNoteSteps.java

```diff
@@ -89,8 +87,7 @@ public void addVisitNote() {
 
     @Then("the system saves the note into visit note table")
     public void systemAddsVisitNote() {
-        assertEquals(DIAGNOSIS_PRIMARY, visitNotePage.primaryDiagnosis());
-        assertEquals(DIAGNOSIS_SECONDARY, visitNotePage.secondaryDiagnosis());
```

Comment:
What is the reason behind replacing these two assertions with just one?

Reply:
Same reason as below, Let me revert these back.